docs/_posts/2020-04-16-release-0-7.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Pomerium 0.7
+date: 2020-4-16
+tags:
+  - release
+  - pomerium
+  - announcement
+author: "Bobby DeSimone"
+---
+
+# Announcing Pomerium 0.7
+
+The 0.7 release of Pomerium lays the groundwork to support rich, dynamic access policies capable of making authorization decisions based on external data-sources from outside your identity provider.
+
+This release includes the following features:
+
+- **Open Policy Agent**: Pomerium now leverages Cloud Native Computing Foundation's [Open Policy Agent](https://www.openpolicyagent.org/) (OPA) as the default policy engine. OPA support in this version clears the pathway to enabling operators to write authorization policy in a high-level, declarative language that promotes safe, performant, fine-grained controls that incorporate data from external records of account. Access control policies will continue to work as before, and are now backed by an even faster evaluation engine.
```

Comment:
>clears the pathway to enabling operators

Might read as a current feature.  Also a bit wordy.  Maybe something like

>OPA support in this version unlocks future capabilities for writing authorization policy in the Rego policy language.  This will promote safe, performant, fine-grained controls to incorporate data from external records of account.

Reply:
Agree, and well put.